### DISCUSSION TITLE
May I ask when Ravyn OS can reach the level of Mac OS 10.15?

### DISCUSSION THREAD

**Discussion by qwt234** [ravynsoft/ravynos#421] [General]
I have been following Ravyn OS and have not seen any official updates on its development in the past two years. Can an official introduce the progress of Ravyn OS here. I am looking forward to using Ravyn OS on my PC!
Thank you for your hard work and dedication. I hope Ravyn OS can end Apple's monopoly position!

**Comment by QW1CKS** [comment]
Pretty sure the project is dead, any repo that doesn't see updates within a month indicates the probability that the devs have left the project and wont make a return. I am looking forward for this project as well.

**Comment by AssassinK786** [comment]
Hello @krypteo @qwt234 
The project is alive and @mszoek is working on the project.
It is hard to get the OS to level of MacOS with limited resources
If you are interested to help for project, pls join discord channel in readme or donate to project via patreon

**Reply by QW1CKS** [reply]
I see, thanks for the clarification @AssassinK786 !

I really do want this project to flourish, and just like you have said, it is indeed hard to get the os to the level of macos with limited resources, it is such a challenging feat.

**Comment by mszoek** [comment]
Not sure why you think it is dead - commits are made pretty much every week. Whenever I have time.

**Reply by QW1CKS** [reply]
Honestly, I have nothing against this project, the slow development made me think like so. I really do hope you guys reach and achieve great feats with this project!
